### PR TITLE
Update KMIP documentation to reflect Vault 1.11 changes.

### PR DESCRIPTION
**Pull request by victorr**
Update KMIP documentation:

Add new subsection describing Vault's KMIP conformance.

Update the list of supported operations.

### CODE REVIEW

**Comment by victorr** [comment]
This is the old `Supportes Operations` subsection.
image

The contents were updated and moved inside subsection `Scopes and Roles`, as the reason for the list is for users to know which operations can be used while configuring roles:

image

**Comment by victorr** [comment]
This is the new `KMIP Conformance` section:

image

For details about conformance, see google doc [Vault’s KMIP Profile Support](https://docs.google.com/document/d/18Q2Yawi7DaR7PwwVxqpzmnkNpGspVDwmjK63krN57OU/edit#).

File: website/content/docs/secrets/kmip.mdx [hunk 1]
```diff
@@ -66,6 +91,43 @@ allowed operations for it.
       Success! Data written to: kmip/scope/my-service/role/admin
     ```
 
+#### Supported KMIP Operations
```
Comment (stevendpclark) [inline website/content/docs/secrets/kmip.mdx:94]: nit: Should this really be underneath Scopes And Roles? I would have thought it better under the Usage header directly? I.E. 

```suggestion
### Supported KMIP Operations
```

**Review by stevendpclark — APPROVED** [review]
Small nit but this lgtm!

File: website/content/docs/secrets/kmip.mdx (hunk 1 above)
Comment (victorr) [reply website/content/docs/secrets/kmip.mdx:94]: Yes, I think that is reasonable.

**Pull request merged**